src/utils.py

```diff
@@ -1261,29 +1416,40 @@ def benchmark(models, num_times, X, y, benchmark, train_size = 0.7, val_size = 0
         )
 
         #recombine train and val for methods that don't use a val set, and for training model after finding markers
```

Comment:
I think these comments need to be removed in the new stuff.

I am a bit confused about the train and validation stuff since some of the methods that we compare to don't make a distinction between train and validation sets when forming the models that are proxies to evaluate the markers. 

Can you explain a bit this change?